db/views/materialized_patient_summaries_v01.sql

```diff
@@ -0,0 +1,118 @@
+       greatest(0,
+                date_part('day', NOW() - next_appointment.scheduled_date))                                            AS days_overdue,
+       next_appointment.id                                                                                            AS next_appointment_id,
+       next_appointment.scheduled_date                                                                                AS next_appointment_scheduled_date,
+       next_appointment.status                                                                                        AS next_appointment_status,
+       next_appointment.remind_on                                                                                     AS next_appointment_remind_on,
+       next_appointment_facility.id                                                                                   AS next_appointment_facility_id,
+       next_appointment_facility.name                                                                                 AS next_appointment_facility_name,
+       next_appointment_facility.facility_type                                                                        AS next_appointment_facility_type,
+       next_appointment_facility.district                                                                             AS next_appointment_district,
+       next_appointment_facility.state                                                                                AS next_appointment_state,
+       (CASE
+            WHEN next_appointment.scheduled_date IS NULL THEN 0
+            WHEN next_appointment.scheduled_date > date_trunc('day', NOW() - interval '30 days') THEN 0
+            WHEN (latest_blood_pressure.systolic >= 180
+                OR latest_blood_pressure.diastolic >= 110) THEN 1
+            WHEN ((mh.prior_heart_attack = 'yes'
+                OR mh.prior_stroke = 'yes')
+                AND (latest_blood_pressure.systolic >= 140
+                    OR latest_blood_pressure.diastolic >= 90)) THEN 1
+            WHEN ((latest_blood_sugar.blood_sugar_type = 'random'
+                AND latest_blood_sugar.blood_sugar_value >= 300)
+                OR (latest_blood_sugar.blood_sugar_type = 'post_prandial'
+                    AND latest_blood_sugar.blood_sugar_value >= 300)
+                OR (latest_blood_sugar.blood_sugar_type = 'fasting'
+                    AND latest_blood_sugar.blood_sugar_value >= 200)
+                OR (latest_blood_sugar.blood_sugar_type = 'hba1c'
+                    AND latest_blood_sugar.blood_sugar_value >= 9.0)) THEN 1
+            ELSE 0
+           END)                                                                                                       AS risk_level,
+       latest_bp_passport.id                                                                                          AS latest_bp_passport_id,
```

Comment:
This is a new column added (`latest_bp_passport.id`) explicitly so we can add a `belongs_to` and do `shortcode` from ruby.